lib/core/di/services_module.dart

```diff
@@ -7,7 +7,9 @@ Future<void> _registerServicesModule() async {
 
```

Comment:
I think we need to remove networking manager as a singleton since we have 1 for each network